website/upgrade-guides/0-15.html.markdown

```diff
@@ -493,3 +494,44 @@ cleanup of obsolete features and improved consistency:
 
     If you are using `-force` in an automated call to `terraform destroy`,
     change to using `-auto-approve` instead.
+
+## Azure Backend Removed Arguments
+
+In an earlier release the `azure` backend changed to remove the `arm_` prefix
+from a number of the configuration arguments:
+
+| Old Name              | New Name          |
+|-----------------------|-------------------|
+| `arm_client_id`       | `client_id`       |
+| `arm_client_secret`   | `client_secret`   |
+| `arm_subscription_id` | `subscription_id` |
+| `arm_tenant_id`       | `tenant_id`       |
+
+The old names were previously deprecated, but we've removed them altogether
+in Terraform v0.15 in order to conclude that deprecation cycle.
+
+If you have a backend configuration using the old names then you may see
+errors like the following when upgrading to Terraform v0.15:
+
+```
+╷
+│ Error: Invalid backend configuration argument
+│
+│ The backend configuration argument "arm_client_id" given on
+│ the command line is not expected for the selected backend type.
```

Comment:
🤔 This seems like an odd choice for example error messages - specifying that the argument was given on the command line.

(this is *just* a comment, you don't need to change this, it just threw me)

Reply:
Fair... it was the case reported in the bug I'm responding to here and so the same thing I was testing locally as part of reproducing that bug. I'm going to run with it for now just to get this published, because the "Invalid backend configuration argument" part was what I was mainly interested in here, but will make a follow-up later (after I retry with a configuration file to get the other error message) if this turns out to be unhelpful.